CHANGELOG

```diff
@@ -1 +1 @@
-[#112] Fix return of the functions get_matched_links(), get_incoming_links(), get_matched_type_template(), get_matched_type() from set to list
```

Comment:
Did you remove the previous changelog message intentionally?

Reply:
Done!